examples/events.rs

```diff
@@ -1,11 +1,19 @@
 extern crate shiplift;
+extern crate tokio;
 
 use shiplift::Docker;
+use tokio::prelude::*;
```

Comment:
When possible I try to avoid wildcards imports because the hide potentially useful documentation. Since these examples files serve as documentation I'd like it to be a bit more clear. What prelude items does this file use/need?

Reply:
Fair enough. I thought this was the recommended way to use Tokio.